### PR TITLE
submitting lesson_1_2

### PR DESCRIPTION
**Pull request by nicoleCodeGirl**

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function AddTodoForm () {
+    return (
+        <>
```
Comment (ethan-carlson) [inline src/AddTodoForm.jsx:3]: react fragment unnecessary if you are returning JSX with one top level tag

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <form>
+            <label>Title</label>
+            <input type="text" id="todoTitle" htmlFor="todoTitle"></input>
```
Comment (ethan-carlson) [inline src/AddTodoForm.jsx:6]: id should be descriptive of the html element, in this case an input.  Eg. `todoInput`

File: src/App.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
-  text-align: center;
+  /* text-align: center; */
```
Comment (ethan-carlson) [inline src/App.css:5]: no need to merge commented out css if it's not serving a purpose

File: src/TodoList.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+    id: 3,
+    title: "watch Udemy videos",
+    }
+  ];
+
+function TodoList() {
+    return (
+        <>
```
Comment (ethan-carlson) [inline src/TodoList.jsx:19]: same comment, no need for the fragment here.

**Review by ethan-carlson — APPROVED** [review]
Nice work!  Let a few small nits to help keep your code looking clean.

File: src/AddTodoForm.jsx (hunk 1 above)
Comment (nicoleCodeGirl) [reply src/AddTodoForm.jsx:3]: Okay I'll remove it. 

function AddTodoForm () {
    return (
        <form>
            <label>Title</label>
            <input type="text" id="todoTitle" htmlFor="todoTitle"></input>
            <button type="submit">Add</button>
        </form>
    )
}

File: src/AddTodoForm.jsx (hunk 2 above)
Comment (nicoleCodeGirl) [reply src/AddTodoForm.jsx:6]: The instructions says:

create a text <input> element with id "todoTitle"

See below:

------------------------------------------------

Screen Shot 2024-03-10 at 12 26 38 AM

File: src/App.css (hunk 3 above)
Comment (nicoleCodeGirl) [reply src/App.css:5]: okay I'll delete the commented out comment.

File: src/TodoList.jsx (hunk 4 above)
Comment (nicoleCodeGirl) [reply src/TodoList.jsx:19]: ok

**Pull request merged**